contracts/Exchange.sol

```diff
@@ -53,6 +53,7 @@ contract Exchange is
     //
 
     struct InternalReplaySwapParams {
+        address trader;
```

Comment:
Although there is no security concern (this struct is only for internal functions), do we have any convention for adding new parameters to the structure?

Reply:
https://github.com/perpetual-protocol/perp-curie-contract/commit/f1ce6bb3f2b004d77257adc041eea2a308d94722